.github/workflows/pr-clean-caches.yml

```diff
@@ -16,9 +17,6 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.ref != 'refs/heads/main' }}
```

Comment:
Why does this workflow run on PR when it has condition like that

Reply:
What do you mean? it's a `!=`

Reply:
Ah I see